<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-QT2UH10T" failures="2" tests="3" name="Default test(failed)" time="0.066" errors="0" timestamp="2023-12-26T09:31:51 IST">
  <testcase classname="testNG_Xml_Use.FAiled_Xml_use" name="Tc2" time="0.012"/>
  <testcase classname="testNG_Xml_Use.FAiled_Xml_use" name="Tc3" time="0.004">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG_Xml_Use.FAiled_Xml_use.Tc3(FAiled_Xml_use.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Tc3 -->
  <testcase classname="testNG_Xml_Use.FAiled_Xml_use" name="Tc5" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG_Xml_Use.FAiled_Xml_use.Tc5(FAiled_Xml_use.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Tc5 -->
  <testcase classname="testNG_Xml_Use.FAiled_Xml_use" name="Tc4" time="0.0">
    <ignored/>
  </testcase> <!-- Tc4 -->
  <testcase classname="testNG_Xml_Use.FAiled_Xml_use" name="Tc1" time="0.0">
    <ignored/>
  </testcase> <!-- Tc1 -->
</testsuite> <!-- Default test(failed) -->
